.github/workflows/release-ep9.yml

```
@@ -0,0 +1,45 @@
+
+on:
+    push:
+        branches:
+            - "develop"
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)